pkg/authorization/cache/policy_cache.go

```diff
@@ -221,3 +245,36 @@ func (c *PolicyCache) ListPolicyBindings(ctx kapi.Context, label labels.Selector
 		return ret, nil
 	}
 }
+
+// Following methods enable the IndexerPolicyCache to implement the ReadOnlyPolicyCache interface
+
+// Policies gives access to a read-only policy interface
+func (this *IndexerPolicyCache) Policies() ReadOnlyPolicyInterface {
```

Comment:
Having some as `this` and some as `c` is going to drive me nuts.  One way or the other and Clayton likes single letters.  Go with `c`